src/ShopRatingServiceBundle/objects/TShopAuskunftAPI.class.php

```diff
@@ -46,6 +48,15 @@ public function TShopAuskunftAPI()
         xml_set_character_data_handler($this->parser, 'tag_contents');
     }
 
+    /**
+     * @deprecated Named constructors are deprecated and will be removed with PHP8. When calling from a parent, please use `parent::__construct` instead.
```

Comment:
"will be removed"?

As this is the step "compatibility with php 8" that sounds like the wrong tense?

Reply:
(Also this commit comes from the issue #652 ?

Reply:
Yes, that commit comes from #93 which is concerned with deprecated named constructors. I built on top of that branch because that PR is also required for PHP8 compatibility